app/models/vm_or_template/operations/relocation.rb

```diff
@@ -23,7 +24,7 @@ def evacuate(options = {})
   end
 
   def validate_evacuate
```

Comment:
yes, both methods should be removed. If the specs fail **then**, we know they are still in use :smile: 

You can also run the whole suite locally
`PARALLEL=true bin/rake test:vmdb` Its *only* 8mins.

You should also grep the codebase for `validate_evacuate` and for `evacuate`. 
For the latter one, it could be tough to spot cases where either the method is called dynamically, like
```ruby
action = 'evacuate'
...
vm.send("validate_" + action)
```
but sometimes you spot something.